src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3068,6 +3068,12 @@ bool AdaptDispatch::WindowManipulation(const DispatchTypes::WindowManipulationTy
     case DispatchTypes::WindowManipulationType::ResizeWindowInCharacters:
         _api.ResizeWindow(parameter2.value_or(0), parameter1.value_or(0));
         return true;
+    case DispatchTypes::WindowManipulationType::ReportTextSizeInCharacters:
+    {
+        const std::wstring textSize = fmt::format(L"\033[8;{};{}t", _api.GetViewport().height(), _api.GetTextBuffer().GetSize().Width());
+        _api.ReturnResponse(std::wstring_view(textSize));
```

Comment:
Nit: I think that `std::wstring` should automatically cast to `std::wstring_view`, so you shouldn't need an explicit cast on the `textSize` parameter. But with that out the way, you might as well embed the `fmt::format` call directly within the `ReturnResponse` call and avoid the `textSize` variable altogether. I suspect we're doing the same thing in a bunch of other places, though, so it's not a big deal.

Reply:
Addressed.